scripts/generateHtml.js

```diff
@@ -36,7 +36,7 @@ const metaContent = (fileName) => `
 `;
 
 /** skypack production optimized url for itwinui-icons-elements */
```

Comment:
Can you add a link to this comment?

```suggestion
/** skypack production optimized url for itwinui-icons-elements from https://cdn.skypack.dev/@itwin/itwinui-icons-elements@latest */
```